config/admin-functions/show-vx-suite-admin-menu.sh

```diff
@@ -148,13 +151,20 @@ while true; do
 
     generate-key)
       sudo "${VX_FUNCTIONS_ROOT}/generate-key.sh"
+      # Generating a new machine private key necessitates recreating the machine cert
+      sudo "${VX_FUNCTIONS_ROOT}/create-machine-cert.sh"
```

Comment:
Not strictly necessary yet since we aren't actually using the TPM private key generated by `generate-key.sh` to create the machine cert. But didn't want to forget this when we do make that shift